web/app/countrytopos.js

```diff
@@ -413,25 +413,26 @@ exports.addCountryTopos = function(countries) {
     countries['US-NE'] = getState('USA', 'US.NE')
     countries['US-NEISO'] = getStates('USA', ['US.CT', 'US.MA', 'US.ME', 'US.NH', 'US.RI', 'US.VT'])
     // countries['US-NH'] = getState('USA', 'US.NH')
-    countries['US-NJ'] = getState('USA', 'US.NJ')
+    // countries['US-NJ'] = getState('USA', 'US.NJ')
     countries['US-NM'] = getState('USA', 'US.NM')
     countries['US-NV'] = getState('USA', 'US.NV')
     countries['US-NY'] = getState('USA', 'US.NY')
-    countries['US-OH'] = getState('USA', 'US.OH')
+    // countries['US-OH'] = getState('USA', 'US.OH')
     countries['US-OK'] = getState('USA', 'US.OK')
     countries['US-OR'] = getState('USA', 'US.OR')
-    countries['US-PA'] = getState('USA', 'US.PA')
+    // countries['US-PA'] = getState('USA', 'US.PA')
     // countries['US-RI'] = getState('USA', 'US.RI')
+    countries['US-PJM'] = getStates('USA', ['US.DE', 'US.KY', 'US.MD', 'US.NJ', 'US.OH', 'US.PA', 'US.VA', 'US.WV'])
```

Comment:
Looks good. Are you able to start the docker containers and submit a screenshot? Then I don't have to double check myself :)

Reply:
Looking into it now.. I have Docker installed... need to youtube the rest. :D

Reply:
let us know on Slack if you have any issues!